runtime/tests/checker/capability_test.go

```diff
@@ -434,3 +434,20 @@ func TestCheckCapability_check(t *testing.T) {
 		})
 	})
 }
+
+func TestCheckCapability_address(t *testing.T) {
+
+	t.Parallel()
+	t.Run("check address", func(t *testing.T) {
+		checker, err := ParseAndCheckWithPanic(t,
+			`		  			
+				let capability: Capability = panic("")
+				let addr = capability.address
+			`,
+		)
+		require.NoError(t, err)
+
+		addrType := RequireGlobalType(t, checker.Elaboration, "addr")
```

Comment:
I think the issue is, `addr` is a variable but not a type. Using `RequireGlobalValue` would solve the test failure.  
```suggestion
		addrType := RequireGlobalValue(t, checker.Elaboration, "addr")
```

Reply:
@ceelo777 like Supun pointed out here, you want to get the type of the variable declaration, which is a value. 

`RequireGlobalType` is used to the the type introduced by a type declaration, e.g. a struct or resource declaration.